bode/bode/models/task/actions.py

```diff
@@ -53,6 +56,16 @@ def is_interchangable_relation(relation):
                     "status": TaskStatus.INDIRECTLY_DONE.value,
                 }
                 edit_task(str(related_task.id), **inter_task_data)
+            if is_subtask_relation(relation):
+                if related_task.status == TaskStatus.DONE.value:
+                    continue
+                subtask_data = {
+                    "title": related_task.title,
+                    "description": related_task.description,
+                    "due_date": related_task.due_date,
+                    "status": TaskStatus.DONE.value,
```

Comment:
Do we need all of this data to be passed when we only want to change the status? I think it should enough to do this and the same for interchangeable tasks:
```suggestion
                    "status": TaskStatus.DONE.value,
```